### PR TITLE
Update CC to Studio Pro 8.12.0

### PR DESCRIPTION
**Pull request by atennapel**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,8 +18,8 @@ apply plugin: 'org.owasp.dependencycheck'
 
 project.ext {
 
-    CC_VERSION = '8.2.0'
-    MXBUILD_VERSION = '7.23.6.53654'
+    CC_VERSION = '8.3.0'
+    MXBUILD_VERSION = '8.12.0.2541'
 }
```
Comment (reinouts) [inline build.gradle:24]: Can we update the OWASP dependency checker to org.owasp:dependency-check-gradle:6.0.3 as well?

**Comment by reinouts** [comment]
I noticed that the UnitTesting module has been updated to fix security vulnerabilities. However we still use an old version of the module in CC. Shouldn't we update that module as well?

**Comment by reinouts** [comment]
Also I think it is safe to accept PR #95 (link to documentation).

File: build.gradle (hunk 1 above)
Comment (atennapel) [reply build.gradle:24]: Done.

**Comment by atennapel** [comment]
Update the PR.

**Comment by reinouts** [comment]
One more remark: It looks like you forgot to remove the unneeded resources (antisamy) added by the old ObjectHandling module.

**Comment by atennapel** [comment]
Oops, removed now...

**Pull request merged**